### ISSUE TITLE
Login component does not give a clear error on duplicate user creation.

### ISSUE THREAD

**Issue by sajvanwijk** [Ycreak/OpensourceClassicCommentary#155]
There isn't currently a clear message that notifies the user that their chosen username is already taken

**Comment by Ycreak** [comment]
Does Flask not provide such message anymore?

**Comment by sajvanwijk** [comment]
Yes, but it just states: "403: forbidden", which might be unclear. Now, I can have it write an alternate message in case of a 403, as in the most recent commit, but the backend should ideally provide a clear message of course. What to do?

**Comment by Ycreak** [comment]
We must ask @borspp @ppbors what he thinks is best. I think Flask should handle this.

**Comment by Ycreak** [comment]
Has been fixed. Error is now: 

    return make_response("User already exists or server error", 403)
